### PR TITLE
sjn added glia

### PR DESCRIPTION
**Pull request by snewhouse**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+git clone --recursive https://github.com/ekg/glia.git && \
```
Comment (bgruening) [inline recipes/build.sh]: @snewhouse you have commited here two recipes (the same). This one should be removed as well as the recipes/meta.yml

Comment (snewhouse) [inline recipes/build.sh]: @bgruening sorry! should be fixed now. I hope

File: recipes/glia/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+git clone --recursive https://github.com/ekg/glia.git && \
```
Comment (bgruening) [inline recipes/glia/build.sh]: Is this really needed? Afaik conda clones the repo for you. Try just `make`. chmod is also probably not needed.

**Comment by snewhouse** [comment]
ok, so I have rushed into this a bit, purely out of excitement - hopefully this is now the right way....

File: recipes/glia/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  git_rev: 9049ffc
+  git_url: https://github.com/ekg/glia.git
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - cmake
```
Comment (bgruening) [inline recipes/glia/meta.yaml]: As good practice you should add GCC for Linux and llvm for osx.
Thanks!

**Comment by bgruening** [comment]
@snewhouse the error seems to be known: https://github.com/ekg/freebayes/issues/83
Just skip the osx builds for the moment with `skip: True # [osx]`

**Comment by snewhouse** [comment]
Hi, I can build using anaconda3 image from Dockerhub. This is now at https://anaconda.org/sjnewhouse/glia. not sure why tests fail for bioconda pull requests.

**Comment by snewhouse** [comment]
running On my local machine (mac osx el capitain)

```bash
docker run \
-v "/Users/sjnewhouse/Google Drive/BIOCONDA/bioconda-recipes":/bioconda-recipes \
-t bioconda/bioconda-builder \
--packages glia --env-matrix /bioconda-recipes/scripts/env_matrix.yml
```

output....

```bash
Collecting git+https://github.com/bioconda/conda-build.git@reintroduce-skip-existing
  Cloning https://github.com/bioconda/conda-build.git (to reintroduce-skip-existing) to /tmp/pip-atx70qzk-build
Requirement already satisfied (use --upgrade to upgrade): conda in /anaconda/lib/python3.5/site-packages (from conda-build===1.20.1-fixed-skip-existing-85.g31f1748)
Installing collected packages: conda-build
  Found existing installation: conda-build 1.20.1
    DEPRECATION: Uninstalling a distutils installed project (conda-build) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.
    Uninstalling conda-build-1.20.1:
      Successfully uninstalled conda-build-1.20.1
  Running setup.py install for conda-build ... done
Successfully installed conda-build-1.20.1-fixed-skip-existing-85.g31f1748
You are using pip version 8.1.1, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
gcc (GCC) 4.8.2 20140120 (Red Hat 4.8.2-15)
Copyright (C) 2013 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

ldd (GNU libc) 2.5
Copyright (C) 2006 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
Packages to build
glia
Building/testing subdag 0 of recipes in order:
/bioconda-recipes/recipes/glia
F
======================================================================
FAIL: __main__.test_recipes('/bioconda-recipes/recipes/glia', <__main__.EnvMatrix object at 0x7fe356f87358>)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/anaconda/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/bioconda-recipes/scripts/build-packages.py", line 103, in build_recipe
    assert False, "At least one build of recipe {} failed.".format(recipe)
AssertionError: At least one build of recipe /bioconda-recipes/recipes/glia failed.
-------------------- >> begin captured stdout << ---------------------
Fetching package metadata: ............
Removing old build environment
Removing old work directory
BUILD START: glia-9049ffc-0
Fetching package metadata: ............
Solving package specifications: .........
gcc install OS matches gcc build OS: Skipping post-link portability fixes.
Cloning into bare repository '/anaconda/conda-bld/git_cache/__github.com_ekg_glia.git'...
Cloning into '/anaconda/conda-bld/work'...
done.
Submodule 'bamtools' (git://github.com/pezmaster31/bamtools.git) registered for path 'bamtools'
Submodule 'fastahack' (git://github.com/ekg/fastahack.git) registered for path 'fastahack'
Submodule 'gssw' (https://github.com/ekg/gssw.git) registered for path 'gssw'
Submodule 'rapidjson' (https://github.com/kennytm/rapidjson.git) registered for path 'rapidjson'
Submodule 'vcflib' (git://github.com/ekg/vcflib.git) registered for path 'vcflib'
Cloning into 'bamtools'...
Submodule path 'bamtools': checked out '2d7685d2aeedd11c46ad3bd67886d9ed65c30f3e'
Cloning into 'fastahack'...
Submodule path 'fastahack': checked out 'c68cebb4f2e5d5d2b70cf08fbdf1944e9ab2c2dd'
Cloning into 'gssw'...
Submodule path 'gssw': checked out '5cd667de259db836f98466f376cec9b33da219dd'
Cloning into 'rapidjson'...
Submodule path 'rapidjson': checked out '52d4fc5f074d6b0525d9eb117f69c00b320ca8b0'
Cloning into 'vcflib'...
Submodule path 'vcflib': checked out '1e2c464f4d07053a306ee5023d91bf133b3016da'
Submodule 'fastahack' (https://github.com/ekg/fastahack.git) registered for path 'fastahack'
Submodule 'filevercmp' (https://github.com/ekg/filevercmp.git) registered for path 'filevercmp'
Submodule 'fsom' (https://github.com/ekg/fsom.git) registered for path 'fsom'
Submodule 'intervaltree' (https://github.com/ekg/intervaltree.git) registered for path 'intervaltree'
Submodule 'multichoose' (https://github.com/ekg/multichoose.git) registered for path 'multichoose'
Submodule 'smithwaterman' (https://github.com/ekg/smithwaterman.git) registered for path 'smithwaterman'
Submodule 'tabixpp' (https://github.com/ekg/tabixpp.git) registered for path 'tabixpp'
Cloning into 'fastahack'...
Submodule path 'vcflib/fastahack': checked out 'c68cebb4f2e5d5d2b70cf08fbdf1944e9ab2c2dd'
Cloning into 'filevercmp'...
Submodule path 'vcflib/filevercmp': checked out '378bae779b897f1506043c1b647a5e5f8031ea4b'
Cloning into 'fsom'...
Submodule path 'vcflib/fsom': checked out 'f688ef24bbe230ec46c7be3de8bf5e67fdbc1b4d'
Cloning into 'intervaltree'...
Submodule path 'vcflib/intervaltree': checked out '1290744283cef8076bb8a2968d4899b7228435f4'
Cloning into 'multichoose'...
Submodule path 'vcflib/multichoose': checked out '73d35daa18bf35729b9ba758041a9247a72484a5'
Cloning into 'smithwaterman'...
Submodule path 'vcflib/smithwaterman': checked out 'd9724f156c07cf16d00d251bebc20c6383eb6bde'
Cloning into 'tabixpp'...
Submodule path 'vcflib/tabixpp': checked out 'c2d6c12eb827805fb13db4bab20f74b212b8b6d0'
cd bamtools && mkdir -p build && cd build && cmake .. && make
-- The C compiler identification is GNU 4.8.5
-- The CXX compiler identification is GNU 4.8.5
-- Check for working C compiler: /anaconda/envs/_build/bin/cc
-- Check for working C compiler: /anaconda/envs/_build/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /anaconda/envs/_build/bin/c++
-- Check for working CXX compiler: /anaconda/envs/_build/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /anaconda/conda-bld/work/bamtools/build
make[1]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[2]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target SharedHeaders
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Built target SharedHeaders
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target APIHeaders
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Built target APIHeaders
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target AlgorithmsHeaders
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Built target AlgorithmsHeaders
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target BamTools
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamAlignment.cpp.o
[  1%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamMultiReader.cpp.o
[  2%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamReader.cpp.o
[  3%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamWriter.cpp.o
[  4%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamHeader.cpp.o
[  5%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamProgram.cpp.o
[  6%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamProgramChain.cpp.o
[  7%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamReadGroup.cpp.o
[  8%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamReadGroupDictionary.cpp.o
[  9%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamSequence.cpp.o
[ 10%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamSequenceDictionary.cpp.o
[ 11%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamHeader_p.cpp.o
[ 12%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamMultiReader_p.cpp.o
[ 13%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamRandomAccessController_p.cpp.o
[ 14%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamReader_p.cpp.o
[ 15%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamWriter_p.cpp.o
[ 16%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/index/BamIndexFactory_p.cpp.o
[ 17%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/index/BamStandardIndex_p.cpp.o
[ 18%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/index/BamToolsIndex_p.cpp.o
[ 19%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamDeviceFactory_p.cpp.o
[ 20%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamFile_p.cpp.o
[ 21%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamFtp_p.cpp.o
[ 22%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamHttp_p.cpp.o
[ 23%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamPipe_p.cpp.o
[ 24%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BgzfStream_p.cpp.o
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[2]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[1]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
commit 9049ffc198cb036c024687dec922b576d75269ef
Author: Erik Garrison <erik.garrison@bc.edu>
Date:   Thu Sep 25 12:42:06 2014 -0400

    MIT license




# Not currently on any branch.
nothing to commit, working directory clean



The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    cmake-3.3.1                |                0        13.0 MB
    gmp-5.1.2                  |                2         594 KB
    isl-0.12.2                 |                0         1.1 MB
    mpfr-3.1.2                 |                0         407 KB
    cloog-0.18.0               |                0         617 KB
    mpc-1.0.1                  |                0          61 KB
    gcc-4.8.5                  |                3        65.8 MB
    ------------------------------------------------------------
                                           Total:        81.5 MB

The following NEW packages will be INSTALLED:

    cloog: 0.18.0-0
    cmake: 3.3.1-0
    gcc:   4.8.5-3
    gmp:   5.1.2-2
    isl:   0.12.2-0
    mpc:   1.0.1-0
    mpfr:  3.1.2-0
    zlib:  1.2.8-0

Removing old work directory
checkout: '9049ffc'
==> git log -n1 <==

==> git describe --tags --dirty <==

==> git status <==

Package: glia-9049ffc-0
source tree in: /anaconda/conda-bld/work

warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
Note: checking out '9049ffc'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

HEAD is now at 9049ffc... MIT license
+ make
/anaconda/conda-bld/work/bamtools/src/api/internal/index/BamStandardIndex_p.cpp: In member function ‘void BamTools::Internal::BamStandardIndex::WriteLinearOffsets(const int&, BamTools::Internal::BaiLinearOffsetVector&)’:
/anaconda/conda-bld/work/bamtools/src/api/internal/index/BamStandardIndex_p.cpp:958:89: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ( numBytesWritten != (sizeof(offsetCount) + linearOffsets.size()*sizeof(uint64_t)) )
                                                                                         ^
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp: In member function ‘bool BamTools::Internal::BamHttp::SendGetRequest(size_t)’:
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp:409:62: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ( WriteToSocket(requestHeader.c_str(), headerSize) != headerSize ) {
                                                              ^
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp: In member function ‘bool BamTools::Internal::BamHttp::SendHeadRequest()’:
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp:501:62: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ( WriteToSocket(requestHeader.c_str(), headerSize) != headerSize ) {
                                                              ^
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BgzfStream_p.cpp:20:18: fatal error: zlib.h: No such file or directory
 #include "zlib.h"
                  ^
compilation terminated.
make[3]: *** [src/api/CMakeFiles/BamTools.dir/internal/io/BgzfStream_p.cpp.o] Error 1
make[2]: *** [src/api/CMakeFiles/BamTools.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [libbamtools.a] Error 2
Using Anaconda Cloud api site https://api.anaconda.org
Command failed: /bin/bash -x -e /bioconda-recipes/recipes/glia/build.sh

Fetching package metadata: ............
Removing old build environment
Removing old work directory
BUILD START: glia-9049ffc-0
Fetching package metadata: ............
Solving package specifications: .........
gcc install OS matches gcc build OS: Skipping post-link portability fixes.
Cloning into '/anaconda/conda-bld/work'...
done.
Submodule 'bamtools' (git://github.com/pezmaster31/bamtools.git) registered for path 'bamtools'
Submodule 'fastahack' (git://github.com/ekg/fastahack.git) registered for path 'fastahack'
Submodule 'gssw' (https://github.com/ekg/gssw.git) registered for path 'gssw'
Submodule 'rapidjson' (https://github.com/kennytm/rapidjson.git) registered for path 'rapidjson'
Submodule 'vcflib' (git://github.com/ekg/vcflib.git) registered for path 'vcflib'
Cloning into 'bamtools'...
Submodule path 'bamtools': checked out '2d7685d2aeedd11c46ad3bd67886d9ed65c30f3e'
Cloning into 'fastahack'...
Submodule path 'fastahack': checked out 'c68cebb4f2e5d5d2b70cf08fbdf1944e9ab2c2dd'
Cloning into 'gssw'...
Submodule path 'gssw': checked out '5cd667de259db836f98466f376cec9b33da219dd'
Cloning into 'rapidjson'...
Submodule path 'rapidjson': checked out '52d4fc5f074d6b0525d9eb117f69c00b320ca8b0'
Cloning into 'vcflib'...
Submodule path 'vcflib': checked out '1e2c464f4d07053a306ee5023d91bf133b3016da'
Submodule 'fastahack' (https://github.com/ekg/fastahack.git) registered for path 'fastahack'
Submodule 'filevercmp' (https://github.com/ekg/filevercmp.git) registered for path 'filevercmp'
Submodule 'fsom' (https://github.com/ekg/fsom.git) registered for path 'fsom'
Submodule 'intervaltree' (https://github.com/ekg/intervaltree.git) registered for path 'intervaltree'
Submodule 'multichoose' (https://github.com/ekg/multichoose.git) registered for path 'multichoose'
Submodule 'smithwaterman' (https://github.com/ekg/smithwaterman.git) registered for path 'smithwaterman'
Submodule 'tabixpp' (https://github.com/ekg/tabixpp.git) registered for path 'tabixpp'
Cloning into 'fastahack'...
Submodule path 'vcflib/fastahack': checked out 'c68cebb4f2e5d5d2b70cf08fbdf1944e9ab2c2dd'
Cloning into 'filevercmp'...
Submodule path 'vcflib/filevercmp': checked out '378bae779b897f1506043c1b647a5e5f8031ea4b'
Cloning into 'fsom'...
Submodule path 'vcflib/fsom': checked out 'f688ef24bbe230ec46c7be3de8bf5e67fdbc1b4d'
Cloning into 'intervaltree'...
Submodule path 'vcflib/intervaltree': checked out '1290744283cef8076bb8a2968d4899b7228435f4'
Cloning into 'multichoose'...
Submodule path 'vcflib/multichoose': checked out '73d35daa18bf35729b9ba758041a9247a72484a5'
Cloning into 'smithwaterman'...
Submodule path 'vcflib/smithwaterman': checked out 'd9724f156c07cf16d00d251bebc20c6383eb6bde'
Cloning into 'tabixpp'...
Submodule path 'vcflib/tabixpp': checked out 'c2d6c12eb827805fb13db4bab20f74b212b8b6d0'
cd bamtools && mkdir -p build && cd build && cmake .. && make
-- The C compiler identification is GNU 4.8.5
-- The CXX compiler identification is GNU 4.8.5
-- Check for working C compiler: /anaconda/envs/_build/bin/cc
-- Check for working C compiler: /anaconda/envs/_build/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /anaconda/envs/_build/bin/c++
-- Check for working CXX compiler: /anaconda/envs/_build/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /anaconda/conda-bld/work/bamtools/build
make[1]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[2]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target SharedHeaders
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Built target SharedHeaders
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target APIHeaders
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Built target APIHeaders
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target AlgorithmsHeaders
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Built target AlgorithmsHeaders
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
Scanning dependencies of target BamTools
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[3]: Entering directory `/anaconda/conda-bld/work/bamtools/build'
[  0%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamAlignment.cpp.o
[  1%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamMultiReader.cpp.o
[  2%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamReader.cpp.o
[  3%] Building CXX object src/api/CMakeFiles/BamTools.dir/BamWriter.cpp.o
[  4%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamHeader.cpp.o
[  5%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamProgram.cpp.o
[  6%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamProgramChain.cpp.o
[  7%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamReadGroup.cpp.o
[  8%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamReadGroupDictionary.cpp.o
[  9%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamSequence.cpp.o
[ 10%] Building CXX object src/api/CMakeFiles/BamTools.dir/SamSequenceDictionary.cpp.o
[ 11%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamHeader_p.cpp.o
[ 12%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamMultiReader_p.cpp.o
[ 13%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamRandomAccessController_p.cpp.o
[ 14%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamReader_p.cpp.o
[ 15%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/bam/BamWriter_p.cpp.o
[ 16%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/index/BamIndexFactory_p.cpp.o
[ 17%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/index/BamStandardIndex_p.cpp.o
[ 18%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/index/BamToolsIndex_p.cpp.o
[ 19%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamDeviceFactory_p.cpp.o
[ 20%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamFile_p.cpp.o
[ 21%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamFtp_p.cpp.o
[ 22%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamHttp_p.cpp.o
[ 23%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BamPipe_p.cpp.o
[ 24%] Building CXX object src/api/CMakeFiles/BamTools.dir/internal/io/BgzfStream_p.cpp.o
make[3]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[2]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
make[1]: Leaving directory `/anaconda/conda-bld/work/bamtools/build'
commit 9049ffc198cb036c024687dec922b576d75269ef
Author: Erik Garrison <erik.garrison@bc.edu>
Date:   Thu Sep 25 12:42:06 2014 -0400

    MIT license




# Not currently on any branch.
nothing to commit, working directory clean



The following NEW packages will be INSTALLED:

    cloog: 0.18.0-0
    cmake: 3.3.1-0
    gcc:   4.8.5-3
    gmp:   5.1.2-2
    isl:   0.12.2-0
    mpc:   1.0.1-0
    mpfr:  3.1.2-0
    zlib:  1.2.8-0

Removing old work directory
checkout: '9049ffc'
==> git log -n1 <==

==> git describe --tags --dirty <==

==> git status <==

Package: glia-9049ffc-0
source tree in: /anaconda/conda-bld/work

warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
warning: protocol restrictions not applied to curl redirects because
your curl version is too old (>= 7.19.4)
Note: checking out '9049ffc'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

HEAD is now at 9049ffc... MIT license
+ make
/anaconda/conda-bld/work/bamtools/src/api/internal/index/BamStandardIndex_p.cpp: In member function ‘void BamTools::Internal::BamStandardIndex::WriteLinearOffsets(const int&, BamTools::Internal::BaiLinearOffsetVector&)’:
/anaconda/conda-bld/work/bamtools/src/api/internal/index/BamStandardIndex_p.cpp:958:89: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ( numBytesWritten != (sizeof(offsetCount) + linearOffsets.size()*sizeof(uint64_t)) )
                                                                                         ^
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp: In member function ‘bool BamTools::Internal::BamHttp::SendGetRequest(size_t)’:
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp:409:62: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ( WriteToSocket(requestHeader.c_str(), headerSize) != headerSize ) {
                                                              ^
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp: In member function ‘bool BamTools::Internal::BamHttp::SendHeadRequest()’:
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BamHttp_p.cpp:501:62: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if ( WriteToSocket(requestHeader.c_str(), headerSize) != headerSize ) {
                                                              ^
/anaconda/conda-bld/work/bamtools/src/api/internal/io/BgzfStream_p.cpp:20:18: fatal error: zlib.h: No such file or directory
 #include "zlib.h"
                  ^
compilation terminated.
make[3]: *** [src/api/CMakeFiles/BamTools.dir/internal/io/BgzfStream_p.cpp.o] Error 1
make[2]: *** [src/api/CMakeFiles/BamTools.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [libbamtools.a] Error 2
Using Anaconda Cloud api site https://api.anaconda.org
Command failed: /bin/bash -x -e /bioconda-recipes/recipes/glia/build.sh


--------------------- >> end captured stdout << ----------------------

----------------------------------------------------------------------
Ran 1 test in 181.415s

FAILED (failures=1)
```

**Comment by bgruening** [comment]
It's probably failing because anaconda has zlib installed and we do not. We want to have zlib as strict dependency. Unfortunately, there are some Makefile that makes it hard to include zlib into the build: like this one: https://github.com/bioconda/bioconda-recipes/pull/1294/files

**Comment by bgruening** [comment]
@snewhouse can you rebase this branch against master? we have changed a lot in the build infrastructure.

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**